backend/api/runner_ws.py

```
@@ -149,6 +171,47 @@ async def _registered_runner_token(organization_id: str) -> str | None:
+    state_code: str,
+    detail_text: str,
+) -> None:
+    try:
+        await record_connector_signal_event(
+            organization_id=organization_id,
+            workspace_id=workspace_id,
+            signal_key=signal_key,
+            state_code=state_code,
+            detail_text=detail_text,
+        )
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)